demo/server.js

```diff
@@ -151,3 +152,28 @@ initSqlJs().then(SQL => {
 
   app.listen(port, () => console.log(`Listening on port ${port}`));
 });
+
+ // Add a new accuracy record for a model
+  function addAccuracyRecord(accuracy)
+  {
+   db.run('INSERT INTO accuracy VALUES (?, ?, ?, ?);', [
+     accuracy.transactionHash,
+     accuracy.blockNumber,
+     accuracy.modelId,
+     accuracy.accuracy,
+   ]);
+   fs.writeFile(dbPath, Buffer.from(db.export()), () => { });
+  }
+
+ // Get the accuracy history for a model
+ function getAccuracyHistory(modelId)
+ {
+  const getAccuracyStmt = db.prepare('SELECT * FROM accuracy WHERE model_id == $modelId ORDER BY timestamp;');
```

Comment:
It would also be good to support searching by model address.
The method should return object with camelCase style keys.

Reply:
I can make one more function to search by model address.
for this thing we need to figure out model_id by model address in the model table 
and then the select rows with that model_id in the accuracy table .

We can do this in one SQLite command as well to support searching by model address.

Reply:
You can search by model address in this function too and have the `SELECT` do a `JOIN` and use `OR` in the search.